source/agora/consensus/state/Ledger.d

```diff
@@ -402,8 +402,8 @@ public class Ledger
 
     protected void addValidatedBlock (in Block block) @safe
     {
-        log.info("Beginning externalization of block #{} (previous block signatures: {})",
-            block.header.height, this.last_block.header.validators);
+        log.info("Beginning externalization of block #{} (block signatures: {})",
+            block.header.height, block.header.validators);
```

Comment:
This is already printed later, let's keep the previous block signatures here.

Reply:
Was thinking it was not so interesting now it does not block the next block. However it is still useful info so will revert.

Reply:
Reverted